components/microsemi/A2F200M3F-FGG256I.stanza

```diff
@@ -204,14 +49,14 @@ public pcb-component component :
     [VAREF1    | T[9]   | Right | analog] 
     [SDD0      | R[2]   | Right | analog] 
     [SDD1      | T[12]  | Right | analog] 
-    [ADC[0]    | T[5]   | Right | analog] 
-    [ADC[1]    | R[6]   | Right | analog] 
-    [ADC[2]    | P[7]   | Right | analog] 
-    [ADC[3]    | M[5]   | Right | analog] 
-    [ADC[4]    | M[9]   | Right | analog] 
-    [ADC[5]    | N[9]   | Right | analog] 
-    [ADC[6]    | P[9]   | Right | analog] 
-    [ADC[7]    | R[9]   | Right | analog] 
+    [ADC[0]    | T[5]   | Left | analog] 
```

Comment:
Likewise, when pin directions change, does this indicate a change in the part requirements or a preferential change?

Reply:
Preferential to get the symbols looking better.